components/automate-ui/src/app/modules/user/user-management/user-management.component.spec.ts

```diff
@@ -15,7 +15,12 @@ describe('UserManagementComponent', () => {
 
```

Comment:
Would be nice to have a test that confirms the error pop-up displays when an input is invalid. Not sure if we have prior art for that though. 🤷‍♂